packages/sidenav/src/sidenav-item.ts

```diff
@@ -38,7 +39,7 @@ export class SideNavItem extends LitElement {
     public disabled = false;
 
     @property({ type: Boolean, reflect: true })
-    public expanded = true;
+    public expanded = false;
```

Comment:
This can create some issues when defaulted to `true` if anyone reading this knows of users to whom I might need point this out, let me know. It's a "feature" change, which will amount to a "breaking" change being we're still pre-1.0.0, so I don't expect there to be issues.